credentials/google/google.go

```diff
@@ -143,3 +145,33 @@ func (c *creds) NewWithMode(mode string) (credentials.Bundle, error) {
 
 	return newCreds, nil
 }
+
+// DefaultCredsBuilder encapsulates a Google Default credential that is built using a
+// JSON config.
+type DefaultCredsBuilder struct{}
+
+// BuildCredsBundle returns a default google credential bundle. Currently the JSON
+// config is unused.
+func (d *DefaultCredsBuilder) Build(_ json.RawMessage) (credentials.Bundle, error) {
+	return NewDefaultCredentials(), nil
+}
+
+// Name returns the name associated with GoogleDefaultCredsBuilder i.e. "google_default".
+func (d *DefaultCredsBuilder) Name() string {
+	return "google_default"
+}
+
+// InsecureCredsBuilder encapsulates a insecure credential that is built using a
+// JSON config.
+type InsecureCredsBuilder struct{}
```

Comment:
This belongs in the `credentials/insecure` package instead.

Reply:
Moved to credentials/insecure.